src/uct/api/uct.h

```diff
@@ -2626,6 +2630,40 @@ UCT_INLINE_API ucs_status_t uct_ep_am_short(uct_ep_h ep, uint8_t id, uint64_t he
 }
 
 
+/**
```

Comment:
no need the comment
should just add
```
/**
 * @ingroup UCT_AM
 * @brief
 */
```
the comment is already part of the description for UCT AM Zcopy operation

Reply:
I was wrong, pls ignore my comments